src/Jarvis.Addin.Files/Indexing/FileIndexer.cs

```diff
@@ -36,7 +37,6 @@ public FileIndexer(IEnumerable<IFileIndexSource> sources, IJarvisLog log)
         {
             _log = new LogDecorator("FileIndexer", log);
             _sources = new List<IFileIndexSource>(sources ?? Array.Empty<IFileIndexSource>());
-            _lock = new AsyncReaderWriterLock();
```

Comment:
We don't actually need a reader-writer lock here if we're careful, because we always do a simple read/write of this variable, so we'll always get a valid reference (i.e. there's no way to get a half-baked reference)